components/ProductMatrix.vue

```diff
@@ -1,16 +1,20 @@
 <template>
   <div>
-    <div v-if="error">
-      {{ error }}
-    </div>
-    <div class="product-matrix">
-      <div v-for="card in cards" :key="card.sortKey">
-        <component :is="components[card.type]" :card="card" />
+    <div ref="matrix" class="product-matrix">
+      <div v-for="(card, index) in cards" :key="card.sortKey">
+        <component
+          :is="components[card.type]"
+          v-if="index === 0"
+          ref="card"
```

Comment:
Must be a better way to get the width of the cards, than having a if for the first card.

Reply:
Done 👍